webcompat/static/css/development/components/filter.css

```diff
@@ -61,3 +61,10 @@
   .wc-Filter--close.is-active {
     background-color:var(--wc-state-close);
   }
+  .wc-Filter--appliedlabel{
+    padding-right: 40px;
+    background-image: url(/img/invalid.svg);
```

Comment:
Ah, I see. You're just reusing the red X from the form validation (re: my comment in the bug about the red X). I wonder if we could find a nice icon from the same icon font set that we use elsewhere in the site. cc @magsout